src/lib/traitify-ui.js

```diff
@@ -2,10 +2,6 @@ import { h, render } from "preact";
 import Main from "../components/main";
 import Promise from 'promise-polyfill';
 
-if (!window.Promise){
```

Comment:
I am forcibly overriding Promise within the context of our lib so it reacts consistently It is wrapped though so it will not override the window.Promise